File: Android /Android/25:Jan:2017/FishbowlMobileApps/AndroidSDK/FishbowlMobileApps-Jamba-Android-v2.4.2/Android_Raleys/Raleys/src/com/raleys/app/android/models/ShoppingList.java
package com.raleys.app.android.models;

import java.io.Serializable;
import java.util.ArrayList;

public class ShoppingList implements Serializable
{
	private static final long serialVersionUID = -7353555533257212179L;
	public String  listId;
	public int     storeNumber;
	public String  name;
	public String  accountId;
	public String  listType;
	public String  modifiedBy;
	public float   totalPrice;
	public float   totalCrv;
	public long    serverUpdateTime;
	public boolean productListReturned;
	public ArrayList<Product> productList;
}

File: Android /Android/25:Jan:2017/FishbowlMobileApps/AndroidSDK/FishbowlMobileApps-Jamba-Android-v2.4.2/iOS_Raleys/Raley's/Raley's/Classes/Models/ShoppingList.m
//
//  ShoppingList.m
//  Raley's
//
//  Created by Bill Lewis on 12/2/13.
//  Copyright (c) 2013 Raleys. All rights reserved.
//

#import "ShoppingList.h"

@implementation ShoppingList

@synthesize listId;
@synthesize name;
@synthesize storeNumber;
@synthesize accountId;
@synthesize listType;
@synthesize modifiedBy;
@synthesize totalPrice;
@synthesize totalCrv;
@synthesize serverUpdateTime;
@synthesize productListReturned;
@synthesize productList;

- (NSString *)getComponentTypeForArray:(NSString *)propertyName
{
    if([propertyName isEqualToString:@"productList"])
        return @"Product";
    else
        return nil;
}

@end
